Source/Plugins/NetworkLoggerPlugin.swift

```diff
@@ -25,14 +25,15 @@ public final class NetworkLoggerPlugin: PluginType {
 
     public func willSendRequest(_ request: RequestType, target: TargetType) {
         if let request = request as? CustomDebugStringConvertible , cURL {
+            output(request.debugDescription)
```

Comment:
i believe it should be 

output(request.debugDescription, separator, terminator)

to match with the method signature for print. but even that is giving errors… I’ve been trying to debug it but  Im not sure if the handling of variadic parameters has changed in swift3… or if its a bug in Xcode8… 



> On Sep 8, 2016, at 1:28 PM, Andrew Breckenridge <notifications@github.com> wrote:
> 
> In Source/Plugins/NetworkLoggerPlugin.swift <https://github.com/Moya/Moya/pull/608#discussion_r78082606>:
> 
> > @@ -25,14 +25,15 @@ public final class NetworkLoggerPlugin: PluginType {
> >  
> >      public func willSendRequest(_ request: RequestType, target: TargetType) {
> >          if let request = request as? CustomDebugStringConvertible , cURL {
> > +            output(request.debugDescription)
> @sunshinejr <https://github.com/sunshinejr> can you explain this line? It's causing an error for me. Is it a typo?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/Moya/Moya/pull/608/files/3e19a97dd45833f625df9f62208b0519f1da2ae1..ee96e1b44a939e0229b43e5c88d74562def93a1c#r78082606>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABf5_q9CR_DJOb7IloiP2isaIhukSf_7ks5qoG_cgaJpZM4J4GFh>.
>